Bigtable/src/Admin/V2/BigtableInstanceAdminClient.php

```diff
@@ -31,12 +31,24 @@
 namespace Google\Cloud\Bigtable\Admin\V2;
 
 use Google\Cloud\Bigtable\Admin\V2\Gapic\BigtableInstanceAdminGapicClient;
+use Google\Cloud\Bigtable\EmulatorSupportTrait;
 
 /**
  * {@inheritdoc}
  */
 class BigtableInstanceAdminClient extends BigtableInstanceAdminGapicClient
 {
-    // This class is intentionally empty, and is intended to hold manual
-    // additions to the generated {@see BigtableInstanceAdminGapicClient} class.
+    use EmulatorSupportTrait;
+
+    /**
```

Comment:
This should be:

```
/**
 * {@inheritDoc}
 */
```

Otherwise the upstream config options won't be displayed.

Reply:
Add `{@inheritDoc}` for constructor? I've tried this, but the doc generator complains: `Inherit Doc tag ({@inheritdoc}) is not supported for reflector type phpDocumentor\Reflection\ClassReflector\MethodReflector`.
I see the only way to show those options: just copy doc block from parent method. Will that do?

Reply:
Ah, I didn't realize that.

Let's use [`modifyClientOptions()`](https://github.com/googleapis/gax-php/blob/master/src/GapicClientTrait.php#L637-L640) instead. In fact, you could simply rename the `setEmulatorOptions` method in your trait and make it accept the argument by reference and it should just work.